docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+services:
+    read-the-docs-builder:
+        build:
+            context: .
+        volumes:
+            - ./:/workspace
+        command: python -m sphinx -T -W --keep-going -b html -d _build/doctrees -D language=en . _build/html
```

Comment:
Maybe also add a linebreak at the end of this file?

Reply:
Added